tests/components/media_source/test_init.py

```diff
@@ -59,14 +64,34 @@ async def test_async_browse_media(hass):
 
 async def test_async_resolve_media(hass):
     """Test browse media."""
+    await async_process_ha_core_config(hass, {})
+    await hass.async_block_till_done()
```

Comment:
Will this have an impact if we run tests in Docker vs bare metal ? Better to just have the test HASS instance by default point at a stub media dir inside the testing config folder.